src/profilers.hpp

```diff
@@ -32,6 +32,7 @@ namespace aperf {
       std::string name;
       std::vector<std::unique_ptr<Requirement>> requirements;
       std::future<int> process;
+      std::map<const int, std::string> errorMessages;
```

Comment:
If you decide to keep the ```handle_errors``` method, use ```std::unordered_map```, it's faster for this use case because you don't need sorting :)

Reply:
Done :+1: